### PR TITLE
refactor(combo, select): update indigo colors

### LINKED ISSUE #301: Indigo: Update select theme

### LINKED ISSUE #302: Indigo: Update combo theme

### PR DESCRIPTION
**Pull request by simeonoff**
Closes #302 
Closes #301

### CODE REVIEW

File: sass/themes/schemas/components/light/_select.scss [hunk 1]
```diff
@@ -151,12 +151,18 @@ $bootstrap-select: extend(
 
 /// Generates an indigo select schema.
 /// @type {Map}
+/// @prop {Map} toggle-button-background [trasnparent] - The select toggle button background color.
+/// @prop {Map} toggle-button-background-focus [transparent] - The select toggle button background color when the select is focused.
+/// @prop {Map} toggle-button-background-focus--border [color: ('gray', 300)] The select toggle button background color when the select is focused.
```
Comment (desig9stein) [inline sass/themes/schemas/components/light/_select.scss:156]: The value should be transparent

File: sass/themes/schemas/components/light/_combo.scss [hunk 2]
```diff
@@ -279,12 +279,25 @@ $bootstrap-combo: extend(
...
+/// @prop {Map} toggle-button-foreground [color: ('gray', 700)] - The combo toggle button foreground color.
+/// @prop {Map} toggle-button-foreground-focus [color: ('gray', 700)] - The combo toggle button foreground color when the combo is focused.
+/// @prop {Map} toggle-button-foreground-filled [color: ('gray', 900)] - The combo toggle button foreground color when the combo is focused.
 /// @prop {Map} toggle-button-foreground-disabled [color: ('gray', 300)] - The combo toggle button foreground color when the combo is disabled.
 /// @prop {Color} toggle-button-background-disabled [transparent] - The combo toggle button background color when the combo is disabled.
 /// @requires {Map} $light-combo
 $indigo-combo: extend(
     $light-combo,
     (
+        empty-list-placeholder-color: (
```
Comment (desig9stein) [inline sass/themes/schemas/components/light/_combo.scss:295]: empty-list-placeholder-color is not documented

File: sass/themes/schemas/components/light/_combo.scss [hunk 3]
```diff
@@ -279,12 +279,25 @@ $bootstrap-combo: extend(
...
 /// @prop {Map} clear-button-foreground [color: ('gray', 600)] - The combo toggle clear foreground color.
 /// @prop {Map} clear-button-foreground-focus [color: ('gray', 600)] - The combo clear button foreground color when the combo is focused.
+/// @prop {Map} toggle-button-background [color: transparent] - The combo toggle button background color.
+/// @prop {Map} toggle-button-background-focus [transparent] - The combo toggle button background color when the combo is focused.
+/// @prop {Map} toggle-button-background-focus--border [transparent] The combo toggle button background color when the combo is focused.
+/// @prop {Map} toggle-button-background-disabled [transparent] - The combo toggle button background color when the combo is disabled.
+/// @prop {Map} toggle-button-foreground [color: ('gray', 700)] - The combo toggle button foreground color.
+/// @prop {Map} toggle-button-foreground-focus [color: ('gray', 700)] - The combo toggle button foreground color when the combo is focused.
+/// @prop {Map} toggle-button-foreground-filled [color: ('gray', 900)] - The combo toggle button foreground color when the combo is focused.
```
Comment (desig9stein) [inline sass/themes/schemas/components/light/_combo.scss:288]: toggle-button-foreground,
toggle-button-foreground-focus,
toggle-button-foreground-filled
Those are not used in this schema, you can remove them

**Comment by simeonoff** [comment]
@desig9stein can you do a second round of reviews, please?

**Review by desig9stein — APPROVED** [review]

**Pull request merged**